devops/ansible/examples/girder-hardened/site.yml

```diff
@@ -0,0 +1,88 @@
+---
+
+- hosts: all
+  vars:
+    girder_update: no
+    girder_force: no
+    girder_hardened: yes
+    nginx_vhosts:
+      - listen: "8080 default_server"
+        ssl_certificate_key: "/etc/ssl/default_server/default_server.key"
+        ssl_certificate: "/etc/ssl/default_server/default_server.pem"
+        extra_parameters: |
+          ssl on;
+          ssl_certificate_key /etc/ssl/default_server/default_server.key;
+          ssl_certificate /etc/ssl/default_server/default_server.pem;
+
+          add_header X-Content-Type-Options nosniff;
+          add_header X-Frame-Options DENY;
+          add_header Content-Security-Policy "style-src 'self' https://fonts.googleapis.com 'sha256-I08hTk63KqFj9IjGtMeORROxIYJYsBEazDOd2hkXWZY='
+                                                                                            'sha256-zB4+dg/iQaGd+pLwCZo03/lJhn/b6UNYzJppzv3EEAc='
+                                                                                            'sha256-h8Ck0CS7cKiFtPMm/8DjdkPXgP8gxqL7G997urcLS28='
+                                                                                            'sha256-MGNMaa1KDEaQ0N9k9XVIdXWtkb9PIkCbTCFGKcQTs7Q=';
+                                              font-src 'self' data: https://fonts.gstatic.com;
```

Comment:
JSONEditor is responsible for the `data:` directive I believe, it's worth looking into if this can be mitigated.